packages/fxa-admin-panel/src/components/AccountSearch/index.test.tsx

```diff
@@ -141,18 +141,158 @@ function exampleBounceMutationResponse(email: string): MockedResponse {
   };
 }
 
+const MinimalAccountResponse = (testEmail: string) => ({
+  data: {
+    accountByEmail: {
+      uid: '123',
+      createdAt: 1658534643990,
+      disabledAt: null,
+      lockedAt: null,
+      emails: [
+        {
+          email: testEmail,
+          isVerified: true,
+          isPrimary: true,
+          createdAt: 1658534643990,
+        },
+      ],
+      emailBounces: [],
+      securityEvents: [],
+      totp: [],
+      recoveryKeys: [],
+      linkedAccounts: [],
+      attachedClients: [],
+      subscriptions: [],
+    },
+  },
+});
+
+const nextTick = (t?: number) => new Promise((r) => setTimeout(r, t || 0));
+
 beforeEach(() => {
   jest.spyOn(window, 'confirm').mockImplementation(() => true);
   testEmail = chance.email();
 });
 
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
 it('renders without imploding', () => {
   const renderResult = render(<AccountSearch />);
   const getByTestId = renderResult.getByTestId;
 
   expect(getByTestId('search-form')).toBeInTheDocument();
 });
 
+it('calls account search', async () => {
+  let calledAccountSearch = false;
+  const mocks = [
+    {
+      request: {
+        query: GET_ACCOUNT_BY_EMAIL,
+        variables: {
+          email: testEmail,
+          autoCompleted: false,
+        },
+      },
+      result: () => {
+        calledAccountSearch = true;
+        return MinimalAccountResponse(testEmail);
+      },
+    },
+  ];
+
+  const renderResult = render(
+    <MockedProvider mocks={mocks} addTypename={false}>
+      <AccountSearch />
+    </MockedProvider>
+  );
+
+  await act(async () => {
+    fireEvent.change(renderResult.getByTestId('email-input'), {
+      target: { value: testEmail },
+    });
+    fireEvent.blur(renderResult.getByTestId('email-input'));
+  });
+  await nextTick();
+
+  await act(async () => {
+    fireEvent.click(renderResult.getByTestId('search-button'));
+  });
+  await nextTick();
```

Comment:
hmm I thought the whole point of wrapping code that affects rendering in `act` was so that we can await the rendering to be completed before we call `expect`.  I assume it was not working and you added `nextTick` as a result?  But if `await act` doesn't I'd think we'd have more test failures.